legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/gitlab/auth/TestGitLabOidcSession.java

```diff
@@ -14,6 +14,9 @@
 
 package org.finos.legend.sdlc.server.gitlab.auth;
 
+import org.finos.legend.sdlc.server.gitlab.mode.GitLabMode;
+import org.junit.Assert;
+import org.junit.Test;
```

Comment:
If the rest of the file didn't change, the imports probably shouldn't change either.